spec.html

```diff
@@ -7890,12 +7890,25 @@ <h1>FunctionDeclarationInstantiation ( _func_, _argumentsList_ )</h1>
   <!-- es6num="9.3" -->
   <emu-clause id="sec-built-in-function-objects">
     <h1>Built-in Function Objects</h1>
```

Comment:
This should be titled "Standard Built-in  Function Objects" as these rules are not intended to be requirements on non-standard built-ins provided by implementations.  The "standard" qualification was apparently lost when this clause was split into two separately titled subclasses.

Reply:
> The "standard" qualification was apparently lost when this clause was split into two separately titled subclasses.

The current title (without "Standard") dates back to ES6 rev 12. The clause does not appear to be the result of a split, and rev 11 does not have a clause titled "Standard Built-in Function Objects".

Reply:
So what's the verdict on this one? I'm happy to add "Standard" if appropriate.

Reply:
Oh, I totally mis-read what this PR was doing.  I now understand and think it is a very bad change.  It complexly breaks the purpose of Clause 17 in relationship to the subsequent clauses and puts stuff in clause 9.3 that doesn't belong there. 

I'll say more, back in #539